addon/components/uni-multi-selector.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/uni-multi-selector';
+
+const { Component, isEmpty } = Ember;
+
+export default Component.extend({
+  layout,
+
+  options: [],
+  name: null,
+  groupValue: null,
+
+  init() {
+    this._super(...arguments);
+
+    if (isEmpty(this.get('groupValue'))) {
+      this.set('groupValue', this.get('options')[0].value);
```

Comment:
The difference is just that, if it works, it uses the Ember Data's```get``` to fetch the final value